bcda/models/models.go

```diff
@@ -340,3 +349,9 @@ type jobEnqueueArgs struct {
 	// TODO: remove `Encrypt` when file encryption disable functionality is ready to be deprecated
 	Encrypt bool
 }
+
+type EncryptionKey struct {
+	gorm.Model
+	Body     string `json:"body"`
+	SystemID uint   `json:"system_id"`
```

Comment:
Need to add an FK reference to System here. It would be another field in the struct (above the SystemID), like so:
	System         models.System 
 `gorm:"foreignkey:SystemID;association_foreignkey:ID"` 

See how auth.Token references ACO as a foreign key; also Job and JobKey